pkg/plugins/policies/meshratelimit/plugin/v1alpha1/testdata/gateway_basic_routes.golden.yaml

```diff
@@ -0,0 +1,48 @@
+name: gateway:HTTP:8080
+requestHeadersToRemove:
+- x-kuma-tags
+validateClusters: false
+virtualHosts:
+- domains:
+  - '*'
+  name: '*'
+  routes:
+  - match:
+      path: /
+    route:
+      weightedClusters:
+        clusters:
+        - name: some-service-21868a8801354f8d
+          requestHeadersToAdd:
+          - header:
+              key: x-kuma-tags
+              value: '&kuma.io/service=gateway&'
+          weight: 1
+        totalWeight: 1
+    typedPerFilterConfig:
+      envoy.filters.http.local_ratelimit:
+        '@type': type.googleapis.com/envoy.extensions.filters.http.local_ratelimit.v3.LocalRateLimit
+        filterEnabled:
+          defaultValue:
+            numerator: 100
+          runtimeKey: local_rate_limit_enabled
+        filterEnforced:
+          defaultValue:
+            numerator: 100
+          runtimeKey: local_rate_limit_enforced
+        responseHeadersToAdd:
+        - append: true
+          header:
+            key: header
+            value: test-value
+        - append: false
+          header:
+            key: other-header
+            value: other-value
```

Comment:
nitpick: at first I thought this is something unrelated, these headers could be named something like `x-kuma-rate-limited` to avoid confusion.